lhotse/cut.py

```diff
@@ -477,7 +502,16 @@ def load_features(self, root_dir: Optional[Pathlike] = None) -> np.ndarray:
             )
         return mixer.mixed_feats
 
-    # TODO: load_audio()
+    def load_audio(self,
+                   recording_set: Optional[RecordingSet] = None,
+                   root_dir: Optional[Pathlike] = None,
+    ) -> List:
+        """Loads the audios of the source cuts and put them into a list."""
+        cuts = [track.cut for track in self.tracks]
+        audios = []
+        for cut in cuts:
+            audios.append(cut.load_audio(recording_set, root_dir))
```

Comment:
I'm not sure if this is right - I think it does not respect the `offset` parameter encoded in the `MixTrack` object that holds the cut, so the returned audio could be in a wrong time span. I saw that you have a test for that but maybe you could extend it to explicitly check such case.